app/main/views/users.py

```diff
@@ -120,28 +120,34 @@ def download_supplier_user_research_report(framework_slug):
 
 
 @main.route('/users/download/buyers', methods=['GET'])
-@role_required('admin-framework-manager', 'admin')
+@role_required('admin-framework-manager')
 def download_buyers():
-    """Either download a list of all buyers (for framework manager) or user research buyers (for admin users)."""
+    """Download a list of all buyers"""
     download_filename = "all-buyers-on-{}.csv".format(datetime.utcnow().strftime('%Y-%m-%d-at-%H-%M-%S'))
     users = data_api_client.find_users_iter(role="buyer")
 
-    if current_user.role == 'admin':
-        # Overwrite the above values for admin specific user research csv
-        download_filename = "user-research-buyers-on-{}.csv".format(datetime.utcnow().strftime('%Y-%m-%d-at-%H-%M-%S'))
-        users = filter(lambda i: i['userResearchOptedIn'], users)
+    return Response(
+        generate_user_csv(users),
+        mimetype='text/csv',
+        headers={
+            "Content-Disposition": "attachment;filename={}".format(download_filename),
+            "Content-Type": "text/csv; header=present"
+        }
+    )
 
-    header_row = ("email address", "name")
-    user_attributes = ("emailAddress", "name")
 
-    def rows_iter():
-        """Iterator yielding header then rows."""
-        yield header_row
-        for user in sorted(users, key=lambda user: user["name"]):
-            yield (user.get(field_name, "") for field_name in user_attributes)
+@main.route('/users/download/buyers/user-research', methods=['GET'])
+@role_required('admin-framework-manager')
+def download_buyers_for_user_research():
+    """Download a list of buyers who have opted in to user research."""
+    users = data_api_client.find_users_iter(role="buyer")
+    # TODO: add param to API endpoint to filter by userResearchOptedIn
```

Comment:
Can probs drop this comment if we're having  separate view

Reply:
This is meant for the actual API - `find_users_iter(role='buyer', user_research_opted_in=True)` would be a nice query to have, to avoid the expensive filter on the frontend. However I forgot to capture it in a tech debt ticket so thanks for reminding me 😸

Reply:
https://trello.com/c/Vgh8eNG8/814-api-findusers-view-should-allow-filtering-by-userresearchoptedin